src/modules/expo/expo-item.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const ExpoItem = (props: any): React.ReactElement => {
+  const expo = props.ExpoI.attributes.real;
+  return (
+    <Link
+      to={`/expos/${props.ExpoI.attributes.slug}`}>
```

Comment:
Esto en una sola linea continua ya que solo hay una sola propiedad que es el `to`

Reply:
listo